src/main/kotlin/org/jetbrains/research/libsl/nodes/statements.kt

```diff
@@ -20,4 +70,32 @@ data class Action(
         append(args.joinToString(separator = ", "))
         append(");")
     }
-}
\ No newline at end of file
+}
+
+data class Proc(
+    val name: String,
+    val arguments: MutableList<Expression> = mutableListOf()
+) : Statement() {
+    override fun dumpToString(): String = buildString {
+        append("${BackticksPolitics.forIdentifier(name)}(")
+        val args = arguments.map { it.dumpToString() }.toMutableList()
+        append(args.joinToString(separator = ", "))
+        append(");")
+    }
+}
+
+data class ExpressionStatement(
```

Comment:
Could you give an example when multiple expressions should be united to a one `ExpressionStatement` class?

Reply:
Fixed this, now there is only one expression